packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -2,12 +2,13 @@
   <div
     class="dropdown-wrapper"
     :class="{ open }"
+    @focusout="close(true)"
```

Comment:
The `close` method will only really close when the arg is false or the device is mobile, and it will never happen here.